### ISSUE TITLE
KSQL should support IN operator

### ISSUE THREAD

**Issue by rmoff** [confluentinc/ksql#2178]
```sql
SELECT TYPE, HOST, MESSAGE, SEVERITY, TAG, FACILITY, REMOTEADDRESS, DATE
FROM SYSLOG_NEW
WHERE HOST IN ('rpi-02','rpi-03','proxmox01')
LIMIT 1;

Failed to generate code for SqlPredicate.filterExpression: (SYSLOG_NEW.HOST IN ('rpi-02', 'rpi-03', 'proxmox01'))schema:org.apache.kafka.connect.data.SchemaBuilder@f680f10isWindowedKey:false
Caused by: not yet implemented: io.confluent.ksql.codegen.SqlToJavaVisitor$Formatter.visitInPredicate
```

Current workaround is to use multiple `OR` matches which is kinda tedious, and `IN` is a very standard operator that people would expect KSQL to support. 

```sql
SELECT TYPE, HOST, MESSAGE, SEVERITY, TAG, FACILITY, REMOTEADDRESS, DATE
FROM SYSLOG_NEW
WHERE HOST = 'rpi-02' OR HOST = 'rpi-03' OR HOST = 'proxmox01'
LIMIT 1;

RFC3164 | proxmox01 | [51847686.416016] audit_printk_skb: 348 callbacks suppressed | null | kernel | 0 | 192.168.10.250 | 1541773381000
```

**Comment by sbouchex** [comment]
Same as #6409 or #6475 ?

**Comment by mjsax** [comment]
The `IN` operator was added in the meantime. Closing this ticket.
